packages/lib-classifier/src/components/Classifier/Classifier.js

```diff
@@ -37,6 +37,7 @@ export default function Classifier({
   useEffect(function onURLChange() {
     const { workflows } = classifierStore
     if (workflowID) {
+      console.log('starting new subject queue', { workflowID, subjectSetID, subjectID })
```

Comment:
```suggestion
      // console.log('starting new subject queue', { workflowID, subjectSetID, subjectID })
```

Console logs in this PR are helpful for classifier debugging, but the ones that log data will be noisy if these are committed to master. I'm suggesting commenting them out until they're specifically used for debugging locally.

Reply:
Console logs are also recorded in Sentry. I've added these so that we can better use Sentry to debug problems on Davy Notebooks and HMS NHS. 

I'd prefer to leave them in until we're sure that those projects are working properly.